internal/proxy/meta_cache.go

```diff
@@ -186,6 +189,71 @@ func (m *MetaCache) GetCollectionSchema(ctx context.Context, collectionName stri
 	return collInfo.schema, nil
 }
 
+func (m *MetaCache) IsCollectionLoaded(ctx context.Context, msgID UniqueID, collectionName string) error {
+	m.mu.RLock()
+	collInfo, ok := m.collInfo[collectionName]
+	if !ok {
+		m.mu.RUnlock()
+		return fmt.Errorf("cannot find collection info in MetaCache, collectionName = %s", collectionName)
+	}
+
+	if !collInfo.isLoaded {
+		m.mu.RUnlock()
+		// check if collection was loaded
+		showResp, err := m.queryCoord.ShowCollections(ctx, &querypb.ShowCollectionsRequest{
+			Base: &commonpb.MsgBase{
+				MsgType:  commonpb.MsgType_ShowCollections,
+				MsgID:    msgID,
+				SourceID: Params.ProxyCfg.ProxyID,
+			},
+		})
+		if err != nil {
+			return err
+		}
+		if showResp.Status.ErrorCode != commonpb.ErrorCode_Success {
+			return errors.New(showResp.Status.Reason)
+		}
+		log.Debug("QueryCoord show collections",
+			zap.Any("collID", collInfo.collID),
```

Comment:
Please use `zap.Int64`, try to avoid using `zap.Any`.

Reply:
fixed